### PR TITLE
Bug 1907299: Add badge for OCS Advanced Subscription

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2020-12-09 20-44-12
Screenshot from 2020-12-14 12-06-15

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/subscription-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          'ceph-storage-plugin~This is an Advanced subscription feature. It requires Advanced Edition subscription. Please contact the account team for more information.',
+        )}
+      >
+        <Label
+          icon={<InfoCircleIcon />}
+          color="purple"
+          variant="filled"
+          className="ceph__subscription"
+        >
+          Advanced Subscription
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/subscription-icon.tsx:23]:
```suggestion
          t(Advanced Subscription)
```

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx [hunk 2]
```diff
@@ -157,7 +158,7 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
           </FormGroup>
           <FormGroup
             fieldId="advanced-encryption-options"
-            label={t('ceph-storage-plugin~Connection settings')}
+            label={<AdvancedSubscription prefix={t('ceph-storage-plugin~Connection settings')} />}
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx:161]: I think this should be in front of storage class encryption checkbox?

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx:161]: Not according to the designs.

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx:161]: @yuvalgalanti Can you check once I think it should be in the checkbox.

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx:161]: Checked with @yuvalgalanti offline and he on moving to `storage class encryption` checkbox

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx:161]: Moving next to `Storage Class Encryption`.

**Comment by bipuladh** [comment]
/remove-component/core
/remove-component/dashboard
/remove-component/dev-console
/remove-component/git-service
/remove-component/insights
/remove-component/knative
/remove-component/kubevirt
/remove-component/lso
/remove-component/metal3
/remove-component/network-attachment-definition
/remove-component/noobaa
/remove-component/olm
/remove-component/sdk
/remove-component/shared

**Pull request merged**